compiler/rustc_passes/src/errors.rs

```diff
@@ -20,6 +20,10 @@ use crate::lang_items::Duplicate;
 #[diag(passes_incorrect_do_not_recommend_location)]
 pub(crate) struct IncorrectDoNotRecommendLocation;
 
+#[derive(LintDiagnostic)]
+#[diag(passes_incorrect_do_not_recmmend_args)]
```

Comment:
```suggestion
#[diag(passes_incorrect_do_not_recommend_args)]
```

here and in the .ftl